client/components/OrderStage/OrderStage.jsx

```diff
@@ -9,7 +9,10 @@ function OrderStage(props) {
 
   const orderStageCallback = useCallback(
     (e) => {
-      const target = e.nativeEvent.path.find((node) => node.tagName === 'BUTTON');
+      let { target } = e.nativeEvent;
```

Comment:
@IgorChaikin Since this part of code is repeated in 5 different files, probably you could move it to some utils function like `getParentTarget` and reuse it, instead of duplicating code